install/kubernetes/helm/istio/charts/grafana/templates/create-custom-resources-job.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if .Values.global.mtls.enabled }}
```

Comment:
Only for mtls ?

Reply:
I mean - the policy must be defined even if global mtls is disabled - that is just the default but there are other ways to enable mtls ( per namespace, etc)

Reply:
@costinm Thanks. Will update it.